Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY glide.lock /go/src/github.com/deis/workflow-cli/
 
 WORKDIR /go/src/github.com/deis/workflow-cli
 
-RUN glide install --strip-vcs --strip-vendor
```

Comment:
`--strip-vcs` is the default in `glide` 0.12+ and it emits a deprecation warning now if you include it.